src/containers/Tablets/Tablets.tsx

```diff
@@ -104,7 +104,6 @@ export const Tablets = ({path, nodeId, className}: TabletsProps) => {
                 onMouseEnter={onShowTooltip}
                 tablet={tabletsToRender[tabletIndex]}
                 key={tabletIndex}
-                className={b('tablet')}
```

Comment:
i believe there are styles for this classname in Tablets.scss?

Reply:
These styles wasn't applied in `Tablet`. So I removed prop but forgot to remove styles themselves.